Doc/library/enum.rst

```diff
@@ -667,28 +667,28 @@ It is also possible to name the combinations::
     ...     X = 1
     ...     RWX = 7
     >>> Perm.RWX
-    <Perm.RWX: 7>
+    Perm.RWX
     >>> ~Perm.RWX
-    <Perm.-8: -8>
+    Perm.-8
```

Comment:
In this case I would prefer `~Perm.RWX` or `Perm(-8)`. But this may be a different issue.

Reply:
Negative values are no longer a thing -- they are converted to their positive equivalent.